_data/stage3.yml

```diff
@@ -91,143 +115,161 @@
     s1.slice(...m1.indices[0]) === &#x22;aaaz&#x22;;
+
     m1.indices[1][0] === 4;
     m1.indices[1][1] === 5;
     s1.slice(...m1.indices[1]) === &#x22;z&#x22;;
+
     m1.indices.groups[&#x22;Z&#x22;][0] === 4;
     m1.indices.groups[&#x22;Z&#x22;][1] === 5;
     s1.slice(...m1.indices.groups[&#x22;Z&#x22;]) === &#x22;z&#x22;;
+
     // capture groups that are not matched return &#x60;undefined&#x60;:
     const m2 = re1.exec(&#x22;xaaay&#x22;);
     m2.indices[1] === undefined;
     m2.indices.groups[&#x22;Z&#x22;] === undefined;
+  has_specification: false
   presented:
-    - date: December&#xA0;2019
+    - date: "December\_2019"
       url: >-
         https://github.com/tc39/notes/blob/master/meetings/2019-12/december-3.md#regexp-match-indices-performance-feedback
-  has_specification: false
-  description: ECMAScript RegExp Match Indicies provide additional information about the start and end indices of captured substrings relative to the start of the input string.
-  authors:
-    - Ron Buckton
-  champions:
-    - Ron Buckton
+  title: RegExp Match Indices
   tests:
     - 'https://github.com/tc39/test262/pull/2309'
-- title: <code>Atomics.waitAsync</code>
-  id: proposal-atomics-wait-async
-  presented:
-    - date: December&#xA0;2019
-      url: >-
-        https://github.com/tc39/notes/blob/master/meetings/2019-12/december-4.md#atomicswaitasync-for-stage-3
-  has_specification: true
-  description: A proposal for an "asynchronous atomic wait" for ECMAScript, primarily for use in agents that are not allowed to block.
+- id: proposal-atomics-wait-async
   authors:
     - Lars Hansen
   champions:
     - Shu-yu Guo
     - Lars Hansen
+  description: >-
+    A proposal for an &ldquo;asynchronous atomic wait&rdquo; for ECMAScript, primarily for
+    use in agents that are not allowed to block.
+  has_specification: true
+  presented:
+    - date: "December\_2019"
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-12/december-4.md#atomicswaitasync-for-stage-3
+  title: <code>Atomics.waitAsync</code>
   tests:
     - 'https://github.com/tc39/test262/issues/2511'
-- title: <code>.at()</code>
-  id: proposal-relative-indexing-method
-  example: |-
-    var list = ["apple", "orange", "banana"];
-    list.at(-1) // banana, using negative index
-    list.at(0)  // apple
-    list.at(1)  // orange
-    list.at(2)  // banana
-  presented:
-    - date: September&#xA0;2020
-      url: item
-  has_specification: true
-  description: A proposal to add a <code>.at()</code> method to all the basic indexable classes (<code>Array</code>, <code>String</code>, <code>TypedArray</code>).
+- id: proposal-relative-indexing-method
   authors:
     - Shu-yu Guo
     - Tab Atkins
   champions:
     - Shu-yu Guo
     - Tab Atkins
+  description: >-
+    A proposal to add a <code>.at()</code> method to all the basic indexable
+    classes (<code>Array</code>, <code>String</code>, <code>TypedArray</code>).
+  example: |-
```

Comment:
This is the polyfill, not the example. The proposal doesn't have an example, and the one I entered manually should be used in this case.